lib/active_admin/filters/resource_extension.rb

```diff
@@ -3,7 +3,7 @@ module Filters
 
     class Disabled < RuntimeError
       def initialize
-        super "Can't remove a filter when filters are disabled. Enable filters with 'config.filters = true'"
+        super "Can't add or remove a filter when filters are disabled. Enable filters with 'config.filters = true'"
```

Comment:
I was thinking we could stop overriding `initialize`, and do `class Disabled < RuntimeError; end`, and then pass the correct message for each of the cases instead?